src/settings/JobProfiles/ViewJobProfile.js

```diff
@@ -482,7 +480,7 @@ export class ViewJobProfile extends Component {
           open={this.state.showRunConfirmation}
           heading={<FormattedMessage id="ui-data-import.modal.jobProfile.run.header" />}
           message={(
-            <SafeHTMLMessage
+            <FormattedMessage
```

Comment:
Why have you replaced `SafeHTMLMessage` with `FormattedMessage`?

Reply:
I see that using of translation values with attributes in translations are no longer supported, but you may still use `react-intl-safe-html` for cases of **b**, **i**, **em**, **strong**, **span**, and **div** _without  attributes_ - this is our case.